airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryUtils.java

```diff
@@ -70,7 +70,7 @@ public class BigQueryUtils {
       DateTimeFormatter.ofPattern("[yyyy][yy]['-']['/']['.'][' '][MMM][MM][M]['-']['/']['.'][' '][dd][d]" +
           "[[' ']['T']HH:mm[':'ss[.][SSSSSS][SSSSS][SSSS][SSS][' '][z][zzz][Z][O][x][XXX][XX][X]]]");
   private static final String USER_AGENT_FORMAT = "%s (GPN: Airbyte)";
-  private static final String CHECK_TEST_DATASET_SUFFIX = "_airbyte_check_stage_tmp_" + System.currentTimeMillis();
```

Comment:
Just curious, wouldn't this timestamp remove cause issues during simultaneous tests execution? Ex. same tests for different  PRs, local run, etc?

Reply:
This shouldn't affect anything since it's implemented back below in `checkHashCreateAndDeleteDatasetRole` and `CHECK_TEST_DATASET_SUFFIX` is only called within that method

Reply:
The reason I moved the timestamp from class instantiation time to test execution time is that it was causing name collision when tests run too fast.